lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -612,6 +612,7 @@ public function migration_ws_validation( array $old_options, array $input ) {
 			return $input;
 		}
 
+		$input['migration_token']    = $this->options->get( 'migration_token' );
```

Comment:
This setting is not typical in that it does not accept an input. The token is generated and then remains in the database (or is received from a [constant](https://auth0.com/docs/cms/wordpress/configuration#php-constant-setting-storage)).

Reply:
what if the value is missing/null? When generated I guess this is not the case. But when taking it from a constant value, are you checking this is not null in the "constants reader" logic?

Reply:
If the value is empty/null, then a token gets generated. 

The constant loading populates `$this->options->get()` so that call catches both.